### PR TITLE
3645 - Dashboard: Hide overview for unpublished regions

### PR DESCRIPTION
**Pull request by sorja**
Note:
Remove unused file left in main PR: src/.../getFaoEstimateViewName.ts

### CODE REVIEW

File: src/client/pages/CountryHome/FraHome/hooks/useSections.tsx [hunk 1]
```diff
@@ -27,7 +28,11 @@ export const useSections = (): Array<Section> => {
 
     if (!cycle) return null
 
-    sections.push({ name: SectionNames.Country.Home.overview, component: Overview })
+    const showOverview = cycle.published || Areas.isISOCountry(countryIso)
```
Comment (sorja) [inline src/client/pages/CountryHome/FraHome/hooks/useSections.tsx]: would it be better to check that it's fra2025 vs published cycle?



```suggestion
const assessmentFra2025 = assessment.props.name = 'fra' && cycle.name = '2025'
const showOverview = assessmentFra2025  || Areas.isISOCountry(countryIso)
```

Comment (sorja) [reply src/client/pages/CountryHome/FraHome/hooks/useSections.tsx]: I am not sure if it matters, only case is that is it possible to publish the data before the dashboard?

Comment (minotogna) [reply src/client/pages/CountryHome/FraHome/hooks/useSections.tsx]: > would it be better to check that it's fra2025 vs published cycle?
> 
> 
> 
> 
> 
> 
> 
> ```suggestion
> 
> const assessmentFra2025 = assessment.props.name = 'fra' && cycle.name = '2025'
> 
> const showOverview = assessmentFra2025  || Areas.isISOCountry(countryIso)
> 
> ```

I think it's good published or country iso

Comment (minotogna) [reply src/client/pages/CountryHome/FraHome/hooks/useSections.tsx]: > I am not sure if it matters, only case is that is it possible to publish the data before the dashboard?

Good point . It shouldn't be possible, but the reality might be different .
It is safer to check fra 2025 explicitly, disregard my previous comment

Comment (sorja) [reply src/client/pages/CountryHome/FraHome/hooks/useSections.tsx]: [1521a9d](https://github.com/openforis/fra-platform/pull/3874/commits/1521a9d32dcd024fd342f2abb4d81828d3de104c)

**Review by minotogna — APPROVED** [review]

**Pull request merged**